packages/database/src/migrations/20210219055420-SamoaHeatmapHomeVillageOfConfirmedCases-modifies-data.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function deleteItemByOneCondition(db, table, condition) {
```

Comment:
I'd be interested to have a discussion about where to define things that will be used in only the `down` migration. Personally I'd be keen to push them to just above the `down`, but I know we have a bit of a convention of this pattern of defining everything here in the migration file. Perhaps you could raise it at dev chat?